agent/service-message-maven-extension/pom.xml

```diff
@@ -12,6 +12,13 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     </properties>
 
+    <repositories>
+        <repository>
+            <id>Develocity extension RC repository</id>
+            <url>https://repo.grdev.net/artifactory/public</url>
+        </repository>
```

Comment:
@clayburn do we still need this?

Reply:
This is not needed anymore, it was from the time when the extension was compiled against the RC version.

Reply:
I've removed it